src/OnboardingSPA/store/selectors.js

```diff
@@ -235,9 +239,21 @@ export function getNextStep( state ) {
 	if ( -1 === currentStepIndex ) {
 		return false; // could not find index
 	}
+	// if current step is theme preview and customDesign is false then skip colors and typography step
+	if ( skipColorsAndTypographyStep( state, '/theme-styles/preview' ) ) {
+		currentStepIndex += 2;
+	}
 	return state.flow.steps.allSteps[ currentStepIndex + 1 ];
 }
 
+export function skipColorsAndTypographyStep( state, pathToSearch ) {
```

Comment:
Can this be renamed to `skipCustomDesignStep`? Assuming there could be more or less design steps added in the future.

Reply:
Removed this function all together while refactoring